tidy3d/components/simulation.py

```diff
@@ -2994,7 +2996,7 @@ def _warn_monitor_simulation_frequency_range(cls, val, values):
                 freqs = np.array(monitor.freqs)
                 if freqs.min() < freq_min or freqs.max() > freq_max:
                     consolidated_logger.warning(
-                        f"monitors[{monitor_index}] contains frequencies "
+                        f"Monitors[{monitor_index}] contains frequencies "
```

Comment:
This was actually not capitalized before it refers to `Simulation.monitors`, but let's put quotes around it: `'monitors[{monitor_index}]'` to make it clear it refers to a variable.